packages/actor-rdf-resolve-quad-pattern-string-source/lib/ActorRdfResolveQuadPatternStringSource.ts

```diff
@@ -79,12 +96,17 @@ export class ActorRdfResolveQuadPatternStringSource extends ActorRdfResolveQuadP
   }
 }
 
-export interface IActorRdfResolveQuadPatternStringSource extends IActorRdfResolveQuadPatternArgs {
+export interface IActorRdfResolveQuadPatternStringSourceArgs extends IActorRdfResolveQuadPatternArgs {
+  /**
+   * The maximum number of entries in the LRU cache, set to 0 to disable.
```

Comment:
```suggestion
   * The maximum number of parsed stores in the LRU cache, set to 0 to disable.
```
(also in the README)

Reply:
Changed this, both in code and readme.